policy-service/pom.xml

```diff
@@ -49,6 +87,10 @@
             <artifactId>caffeine</artifactId>
             <version>${caffeine.version}</version>
         </dependency>
+        <dependency>
```

Comment:
The "spring-boot-starter-data-redis" dependency is redundant remove one of the two.

Reply:
removed